internal/common/config/clientconfig_test.go

```diff
@@ -0,0 +1,85 @@
+package config
+
+import (
+	"reflect"
+	"testing"
+)
+
+// Tests GetClientDisasterConfig too
+func TestGetClientConfig(t *testing.T) {
```

Comment:
how did you do this so fast @lhl2617 😭

Reply:
i am speed

Reply:
![image](https://user-images.githubusercontent.com/33488131/103553485-6a01b800-4ea5-11eb-8980-2af1d8538f2e.png)